docs/kpod-stop.1.md

```diff
@@ -0,0 +1,35 @@
+% kpod(1) kpod-stop - Stop one or more containers
+% Brent Baude
+# kpod-stop "1" "September 2017" "kpod"
+
+## NAME
+kpod stop - Stop one or more containers
+
+## SYNOPSIS
+**kpod** **stop** [*options* [...]] container
+
+## DESCRIPTION
+Kpod stop will stop one or more containers.  You may use container IDs or names as input.  The **--timeout** switch
+allows you to specify the number of seconds to wait before forcibly stopping the container after the stop command
+is issued to the container.  The default is 10 seconds.
```

Comment:
Looks like there is a double space after the period

Reply:
fixed